sdk/src/network-client.ts

```diff
@@ -101,7 +101,7 @@ class AleoNetworkClient {
      */
     async fetchData<Type>(url = "/"): Promise<Type> {
         try {
-            return parseJSON(await this.fetchRaw(url));
+            return retryWithBackoff(() => this.fetchRaw(url).then(parseJSON));
```

Comment:
For the GET, we probably don't want to wrap this directly as whether to retry depends on a lot of different factors for different methods. We should instead wrap the individual method gets themselves with error codes and conditions that are contextual to that.